patternfly/patternfly-elements
# Add large size to health index

## Add large size to health index

* `http://localhost:8000/elements/pfe-health-index/demo/`

### What has changed and why
Summarize files edited as part of this MR along with a brief description of what was changed/why.

* I've added a size attribute which doubles the height and width of each box when set to large (`size="lg"`)

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

1. Pull the branch down and type run npm start in your local environment
2. Navigate to http://localhost:8000/elements/pfe-health-index/demo/ and verify the size variations

#### Browser requirements
Your component should work in all of the following environments:


Your repository infrastructure updates should work for at least:

### Ready-for-merge Checklist



**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
